### ISSUE TITLE
Add priority

### ISSUE THREAD

**Issue by AkashDev23** [Des-Vu-Technologies/TodoApp#10]
Adding priority to tasks helps you focus on what's most important and ensures that high-value tasks are completed first.

**Comment by shankarkharel** [comment]
Absolutely, adding priority to tasks is a vital component of effective task management. By assigning priorities to tasks, can help better allocate time and resources to focus on what's most important and urgent. 

When assigning priorities to tasks, you can use various methods, such as the Eisenhower Matrix (categorizing tasks as urgent/important, important/not urgent, etc.) or a simple numbering system (e.g., 1 for highest priority, 2 for medium, 3 for low).

Remember that priorities can change, so it's essential to periodically review and adjust them as needed. What's high-priority today may not be the same in a week or a month. Flexibility in your task management process is key to staying agile and responsive to changing needs and opportunities.
